content/starter-configs/code-snippets/static-hugo.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Hugo Static Site"
+date: 2019-09-19T11:00:39-04:00
+weight: 2
+---
+
+A Tugboat Preview for a [Hugo static site](https://gohugo.io/) builds the static site, and then serves it. This config
+generates a static site that looks like your `hugo` build on your local development environment, but is accessible via
+the secure Tugboat URL by anyone who has the link; no need for the viewer to have a local environment.
+
+## Configure Tugboat
+
+The Tugboat configuration is managed by a [YAML file](/setting-up-tugboat/create-a-tugboat-config-file/) at
+`.tugboat/config.yml` in the git repository. Here's a basic Hugo static site configuration you can use as a starting
+point, with comments to explain what's going on:
+
+```yaml
+services:
+  # What to call the service hosting the site. Because there is only
+  # one service, it is automatically set as the default service, which
+  # does a few things
+  #   1. Clones the git repository into the service container
+  #   2. Exposes port 80 to the Tugboat HTTP proxy
+  #   3. Routes requests to the preview URL to this service
+  apache:
+    # Use the available version of Apache by not specifying a version
+    image: tugboatqa/httpd
+
+    # A set of commands to run while building this service
+    commands:
+      # Commands that set up the basic preview infrastructure
+      init:
+        # Use curl to download Hugo into the Apache service
+        - curl -Ls https://github.com/gohugoio/hugo/releases/download/v0.58.3/hugo_0.58.3_Linux-64bit.tar.gz | tar -C
+          /usr/local/bin -zxf
+        # Install Hugo
+        - hugo
+        # I don't know what these two things do, and if they're needed for this Hugo starter config. Ben?
```

Comment:
these enable/configure apache's mod_rewrite module, which we use to redirect our old URL structure to the new one. These are safe to remove for generic hugo instructions